.eslintrc.cjs

```diff
@@ -86,6 +86,19 @@ module.exports = {
           }
         ],
 
+        // Check relative import paths are used
+        'no-restricted-imports': [
+          'error',
+          {
+            patterns: [
+              {
+                group: ['../'],
```

Comment:
Nice one. What about sibling imports?

```suggestion
                group: ['./', '../'],
```

Reply:
I thought we use `./` in some places? let me check

Reply:
in 1 place :), I will correct that and add it to the rule

Reply:
done